src/modules/gifmaker/AnimatedGifRecorder/RecorderConf.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace AnimatedGifRecorder
+{
+    /// <summary>
+    /// Summarizes capture session configurations in a single object
+    /// </summary>
+    public class RecorderConf
```

Comment:
Probably just write out the Configuration word, you can use Conf when your writing variables

Reply:
Noted, thank you Alex!